README.md

```diff
@@ -61,6 +60,44 @@ SingleStorePoolDataSource pool = new SingleStorePoolDataSource("jdbc:singlestore
     }
 ```
 
+## Extended types support
+
+Starting with SingleStore database version 8.7.1, extended types metadata is supported. 
+This feature can be enabled using the `enable_extended_types_metadata` session variable. 
+By default, this variable is set to false for backward compatibility, meaning the system will use the standard data type format.
+
+With SingleStore JDBC driver version 1.2.6 and later, you can enable extended types metadata by setting the `enableExtendedDataTypes` parameter to true.
+When enabled, JDBC will handle extended types metadata properly.
+
+**Note: The `enableExtendedDataTypes` JDBC parameter overrides the manually set `enable_extended_types_metadata` session variable.**
```

Comment:
I don't understand what does this mean... Does it refer to the global variable `enable_extended_types_metadata`, or `SET SESSION enable_extended_types_metadata = [ON|OFF]` has no effect when called from the JDBC driver?

Reply:
means JDBC driver on connection establishment always execute query: SET SESSION enable_extended_types_metadata = true/false and will override any previously set.

Reply:
okay, then let's replace "manually set `enable_extended_types_metadata` session variable." with "`enable_extended_types_metadata` server (global) variable"